openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -530,7 +547,7 @@ const EntityPageInfo = ({
       />
       {isAnnouncementDrawerOpen && (
         <AnnouncementDrawer
-          createAnnouncementPermission={createAnnouncementPermission}
+          createAnnouncementPermission={permission?.EditAll}
```

Comment:
```suggestion
          createPermission={permission?.EditAll}
```
Since we are already in context of Announcement